Adding the context around support for 3 integrations (#2505)
hashicorp/web-unified-docs: content/vault/v2.x/content/docs/secrets/pki-external-ca/index.mdx

1. **Automate certificate acquisition** through the ACME protocol.
1. **Cache and distribute certificates** to applications.

> The PKI External CA secrets engine has been tested to work with

Reviewer: ```suggestion
The PKI External CA secrets engine is verified to work with
```
Style correction: write in active voice
